pkg/cmd/issue/edit/edit.go

```diff
@@ -107,8 +117,12 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 			if flags.Changed("add-project") || flags.Changed("remove-project") {
 				opts.Editable.Projects.Edited = true
 			}
-			if flags.Changed("milestone") {
+			if flags.Changed("milestone") || opts.RemoveMilestone {
 				opts.Editable.Milestone.Edited = true
+
+				// Note that when `--remove-milestone` is provided, the value of
+				// `opts.Editable.Milestone.Value` will automatically be empty,
+				// which results in milestone association removal.
```

Comment:
👍 Doing due diligence, this is where edited milestone handling is:
https://github.com/cli/cli/blob/a81a1f7e906ef3666ef3913efd1232afea634bd4/pkg/cmd/pr/shared/editable.go#L190-L200

Reply:
Thanks, and yeah, I've noticed this method. At first, I wanted to expose the `noMilestone` const (i.e., the `none` value) and assign the `opts.Editable.Milestone.Value` with that but then I realized this is meant to be used for the interactive scenario (as a human-readable value rather than an empty string which cannot be clearly displayed in the terminal).

Anyway, if you think it's a good idea I can explicitly refer to this function in the comment block.

Reply:
I've pointed to the `Editable.MilestoneId` in the comments.